pinot-broker/src/test/java/com/linkedin/pinot/broker/queryquota/TableQueryQuotaManagerTest.java

```diff
@@ -51,9 +55,18 @@
   private static String OFFLINE_TABLE_NAME = RAW_TABLE_NAME + "_OFFLINE";
   private static String REALTIME_TABLE_NAME = RAW_TABLE_NAME + "_REALTIME";
 
-
   @BeforeTest
   public void beforeTest() {
+    Enumeration<Logger> loggers = Logger.getRootLogger().getLoggerRepository().getCurrentLoggers();
+    while (loggers.hasMoreElements()) {
+      Logger logger = loggers.nextElement();
+      System.out.println("Log name: " + logger.getName());
+      if (logger.getName().startsWith("com.linkedin.pinot.common.utils") ||
```

Comment:
I think it will be sufficient if we enabled it only for ZkStarter right? Also, can this be added in a common/base class instead of setup for multiple tests? Is logging initialized per test? 
Also, should this be reset in a teardown method?